src/cell.rs

```diff
@@ -234,17 +236,19 @@ impl Cell {
             1 => self.config_frame.as_ptr() as usize,
             _ => self.config_frame.start_paddr(),
         };
-        unsafe { CellConfig::new(&(config_addr as *const HvCellDesc).as_ref().unwrap()) }
+        unsafe { CellConfig::new((config_addr as *const HvCellDesc).as_ref().unwrap()) }
     }
 
     pub fn gpm_activate(&self) {
         unsafe { self.gpm.activate() }
     }
-
+    /// Query an ipa from cell's stage 2 page table to get pa.
     pub fn gpm_query(&self, gpa: GuestPhysAddr) -> usize {
         unsafe { self.gpm.page_table_query(gpa).unwrap().0 }
     }
-
+    /// Map a mem region to a cell. \
```

Comment:
I add some doc annotates to some public functions. I'm confused when there aren't any comments about the function.